### PR TITLE
feat: createdAt to PersistentOrders table

### PR DESCRIPTION
**Pull request by romainbutteaud**
- adding "created_at" to the persistent_orders table for matcha to be able to sort orders by creation date 

# Description

# Testing Instructions

# Checklist

-   [ ] Update [documentation](https://github.com/0xProject/website/blob/development/mdx/api/index.mdx) as needed. **Website Documentation PR:**
-   [x] Prefix PR title with `[WIP]` if necessary.
-   [x] Add tests to cover changes as needed.
-   [x] Test changes on the staging environment with [Matcha API Staging](https://api-staging.matcha.xyz).

    -   [x] SRA/Limit orders
    -   [ ] Swap endpoints
    -   [ ] Meta transaction endpoints
    -   [ ] Depth charts

    For more information see `0x API Matcha smoke test runbook` in Quip.

### CODE REVIEW

**Comment by romainbutteaud** [comment]
deploy staying

**Comment by opaolini** [comment]
deploy staging

File: README.md [hunk 1]
```diff
@@ -108,13 +108,13 @@ To get a local development version of `0x-api` running:
 
 5. Run `docker-compose up` to run the other dependencies required for the API. This uses the local `docker-compose.yml` file. On start-up, the [event-pipeline](https://github.com/0xProject/0x-event-pipeline) container will crash and restart until Postgres is up. If you switch `CHAIN_ID` after a prior run, you will have to `rm -rf 0x_mesh postgres` to delete the volumes containing stale data.
 
-6) Run the database migrations:
+6. Run the database migrations:
 
 ```
-yarn migrate
+yarn db:migrate
```
Comment (xianny) [inline README.md:114]: 👍

**Review by xianny — APPROVED** [review]
Looks good to me!

File: README.md (hunk 1 above)
Comment (romainbutteaud) [reply README.md:114]: forgot to commit `package.json` with this change, added `db:revert` too

**Pull request merged**